source/factory.js

```diff
@@ -1,9 +1,9 @@
-var deepmerge = require("deepmerge");
+const deepmerge = require("deepmerge");
 
-var authTools = require("./auth.js"),
+const authTools = require("./auth.js"),
     urlTools = require("./url.js");
```

Comment:
These should be updated stylistically, to something like this:

```javascript
const authTools = require("./auth.js");
const urlTools = require("./url.js");
```

(no comma-separation of lines when using const)